### ISSUE TITLE
Cannot read properties of undefined (reading 'split')

### ISSUE THREAD

**Issue by marcorossi-cb** [Noovolari/leapp#434]
**Describe the bug**
When attempting to open connection on browser (cmd + click) the above error message is encountered.
This error is shown only for IAM User provider

**Leapp Version**
Leapp Version: 0.18.0

**To Reproduce**
1. Create a new Amazon AWS Session
2. Choose AWS IAM User access method
3. Start connection
4. Cmd + click
5. See error

**Expected behavior**
Open collection in browser

**Screenshots**
![Schermata 2023-06-26 alle 10 55 05](https://github.com/Noovolari/leapp/assets/67049168/d33198bb-fd01-493c-80e5-ad73bed0199f)
![Schermata 2023-06-26 alle 10 55 14](https://github.com/Noovolari/leapp/assets/67049168/fa0470ea-e145-447c-8bfc-508637c73728)

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - OS Version: 12.3
 - Leapp Version: 0.18.0

**Additional context**
Basically I'm trying to open multi connections that come from IAM User or IAM Single Sign-On

**Comment by rick-rtt** [comment]
Hello @marcorossi-cb, thanks for the feedback. At the moment, we don't support opening the AWS web console for AWS IAM User sessions, so the shortcut should be disabled and this behaviour is in fact wrong. However, I tried to replicate the issue on my setup and the function is wrongly activated but it doesn't prompt this error. Can you provide us with the logs? You can find them on macOS here `~/Library/Logs/Leapp/log.electronService.log`. Finally, are you using the multi-console web extension? Thanks again for the feedback!

**Comment by andreacavagna01** [comment]
Is this issue now fixed @marcorossi-cb ?
